spec/factories/users.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :user do
+    name { Faker::Name.name }
+    email { Faker::Internet.email }
+    password_digest { BCrypt::Password.create('password123') }
+
```

Comment:
we can remove extra linespace, try rubocop

Reply:
installed and ran rubocop bhaiya